src/util/KVStore/AMOCommand.cpp

```diff
@@ -0,0 +1,39 @@
+// Copyright 2019 Jack Khuu, Justin Johnson, Lukas Joswiak
+
+#include "AMOCommand.hpp"
+
+#include <string>
+
+#include "Action.hpp"
+
+namespace AMOCommand {
+using std::string;
+
+AMOCommand::AMOCommand(const int &seqNum, const int &key, const string &value,
```

Comment:
you don't need constant references for ints or strings.

Reply:
String is not native to c++ so it should be by ref, but yes for ints